src/ooxml/java/org/apache/poi/xssf/streaming/OpcOutputStream.java

```diff
@@ -0,0 +1,124 @@
+package org.apache.poi.xssf.streaming;
```

Comment:
can you add the apache license to all files?

Reply:
sure. done.